.github/workflows/beta-release.yml

```diff
@@ -37,12 +35,12 @@ jobs:
               run: |
                   echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
 
-            - name: Publish beta release
+            - name: Publish beta snapshot release
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
               run: |
                   git add .
-                  git commit -am "chore: publish beta version"
+                  git commit -am "chore: publish beta snapshot version"
                   yarn prepublishOnly
-                  yarn changeset publish --no-git-tag
+                  yarn changeset publish --no-git-tag --tag beta
```

Comment:
```suggestion
                  yarn changeset publish --no-git-tag --tag beta-${GITHUB_SHA::8}
```

This will append the 8-digit SHA of the merge commit going in to main. this is provided TO changesets and would be in the {tag} position in the template. I tested this in #5447 to make sure the tag would compile correctly. Screenshot of this suggestions output.

<img width="515" alt="Screenshot 2025-05-06 at 5 04 47 PM" src="https://github.com/user-attachments/assets/fb5cbd51-fbb1-4353-82f2-1d5423173939" />

Reply:
applied this to the wrong line. this tag is associated with NPM tags, moving this suggestion up to line 28